Source/build/Prism.Wpf.nuspec

```diff
@@ -1,31 +0,0 @@
-<?xml version="1.0"?>
-<package xmlns="http://schemas.microsoft.com/packaging/2012/06/nuspec.xsd">
-  <metadata>
-    <id>Prism.Wpf</id>
-    <version>$version$</version>
-    <title>Prism 7 for WPF</title>
-    <authors>Brian Lagunas, Bart Lannoeye, Dan Siegel</authors>
-    <owners>Brian Lagunas, Bart Lannoeye, dansiegel</owners>
-    <licenseUrl>https://github.com/PrismLibrary/Prism/blob/master/LICENSE</licenseUrl>
-    <projectUrl>http://prismlibrary.com/</projectUrl>
-    <iconUrl>http://prismlibrary.github.io/images/prism-logo-graphic-128.png</iconUrl>
-    <requireLicenseAcceptance>false</requireLicenseAcceptance>
-    <description>Prism is a fully open source version of the Prism guidance originally produced by Microsoft Patterns &amp; Practices.  Prism provides an implementation of a collection of design patterns that are helpful in writing well structured and maintainable XAML applications, including MVVM, dependency injection, commanding, event aggregation, and more. Prism's core functionality is a shared code base in a Portable Class Library targeting these platforms; WPF, Windows 10 UWP, and Xamarin Forms. Features that need to be platform specific are implemented in the respective libraries for the target platform.
-
-Prism.Wpf helps you more easily design and build rich, flexible, and easy to maintain Windows Presentation Foundation (WPF) desktop applications. This library provides user interface composition as well as modularity support.</description>
-    <summary>Prism libraries related to user interface composition, regions, and modularity for WPF.</summary>
-    <releaseNotes>$releaseNotes$</releaseNotes>
-    <language>en-US</language>
-    <tags>prism mvvm xamarin xaml wpf</tags>
-    <dependencies>
-      <group targetFramework=".NETFramework4.5">
-        <dependency id="CommonServiceLocator" version="2.0.3" />
-        <dependency id="Prism.Core" version="$coreVersion$" />
-      </group>
-    </dependencies>
-  </metadata>
-  <files>
-    <file src="..\Wpf\Prism.Wpf\bin\Release\Prism.Wpf.*" target="lib\net45\" exclude="**\*.pdb" />
-    <file src="..\Wpf\Prism.Wpf\bin\Release\System.Windows.Interactivity.dll" target="lib\net45\" />
```

Comment:
Looks like dropping the nuspec causes #1492

Reply:
the csproj was updated to pack the dll... I'll take a closer look into it.

Reply:
Check this [issue](https://github.com/NuGet/Home/issues/3891). Im not sure if it related to Reference or not, but you can find workaround there.
Or I propose to add PackageReference to one of NuGet package that contains this dll (for example [this one](https://www.nuget.org/packages/Expression.Blend.Sdk.WPF/)).